esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -444,8 +444,22 @@ void EsdlServiceImpl::configureUrlMethod(const char *method, IPropertyTree &entr
 
     Owned<ISmartSocketFactory> sf = createSmartSocketFactory(iplist, true);
 
-    connMap.remove(method);
-    connMap.setValue(method, sf.getClear());
+    try
+    {
+        Owned<ISmartSocketFactory> sf = createSmartSocketFactory(iplist, true);
```

Comment:
Above lines use \"%s\" for method, perhaps match that here just to be consistent